src/main/java/jdbcecomarce/Connections.java

```diff
@@ -0,0 +1,198 @@
+		return con;
+	}
+
+	public static void createTables() throws SQLException {
+		Connection connection = getConnetion1();
+
+		Statement statement = connection.createStatement();
+
+		String query = "DROP TABLE IF EXISTS passengers";
+		String query1 = "DROP TABLE IF EXISTS bookingss";
+		statement.execute(query);
+		statement.execute(query1);
+
+		StringBuffer stringBuffer = new StringBuffer("create table Customers(");
+		stringBuffer.append("CUSTOMERID INT PRIMARY KEY,");
+		stringBuffer.append("NAME VARCHAR(100) NOT NULL, ");
+		stringBuffer.append("AGE INT NOT NULL,");
+		stringBuffer.append("GENDER VARCHAR(20) NOT NULL,");
+		stringBuffer.append("ADDRESS VARCHAR(200) NOT NULL");
+		stringBuffer.append(")");
+
+		statement.execute(stringBuffer.toString());
+
+		StringBuffer stringBuffer1 = new StringBuffer("create table Purchases(");
+		stringBuffer1.append("CUSTOMERID INT NOT NULL,");
+		stringBuffer1.append("PURCHASEID INT PRIMARY KEY,");
+		stringBuffer1.append("QUANTITY INT NOT NULL,");
+		stringBuffer1.append("ITEMPURCHASED VARCHAR(100) NOT NULL,");
+		stringBuffer1.append("PRICE DECIMAL(10,2) NOT NULL,");
+		stringBuffer1.append("DATEOFPURCHASE DATE,");
+		stringBuffer1.append("FOREIGN KEY (customerId) REFERENCES Customers(customerId)");
+		stringBuffer1.append(")");
+
+		statement.execute(stringBuffer1.toString());
+
+		System.out.println("Tables Scussefully Created");
+	}
+
+	public static void insertdataIntoTables(ArrayList<LaptopCustomers> laptopcustomers,
+			ArrayList<LaptopPurchases> laptopPurchases) throws SQLException {
+
+		Connection connection = getConnetion1();
+		String sql = "INSERT INTO customers (customerId, name,  age, gender, address) VALUES (?, ?, ?, ?, ?)";
+		String sql1 = "INSERT INTO PURCHASES (customerId,purchaseId,quantity,itemPurchased,price,dateOfPurchase) VALUES (?, ?, ?, ?, ?,?)";
+		PreparedStatement statement = connection.prepareStatement(sql);
+		PreparedStatement statement1 = connection.prepareStatement(sql1);
+		for (LaptopCustomers customer : laptopcustomers) {
+			statement.setInt(1, customer.getCustomerId());
+			statement.setString(2, customer.getName());
+			statement.setInt(3, customer.getAge());
+			statement.setString(4, customer.getGender());
+			statement.setString(5, customer.getAddress());
+
+			statement.executeUpdate();
+		}
+
+		for (LaptopPurchases purchase : laptopPurchases) {
+			statement1.setInt(1, purchase.getCustomerId());
+			statement1.setInt(2, purchase.getPurchaseId());
+			statement1.setInt(3, purchase.getQuantity());
+			statement1.setString(4, purchase.getItemPurchased());
+			statement1.setDouble(5, purchase.getPrice());
+			statement1.setString(6, purchase.getDateOfPurchase());
+
+			statement1.executeUpdate();
+		}
+
+		System.out.println(" Data Inserted successfully");
+	}
+
+	public static void readTables(int customerId) throws SQLException {
+
+		Connection connection = getConnetion1();
+		String sql = "select * from customers where customerId=?";
```

Comment:
add purchanses information, meaning join tables and get the data

Reply:
done sir